velox/exec/tests/AggregationTest.cpp

```diff
@@ -123,6 +123,22 @@ class AggregationTest : public OperatorTestBase {
     return vectors;
   }
 
+  template <typename T>
+  void testFloatingPointArrayNaNGroupBy() {
```

Comment:
This change is better tested in a unit test inside velox/exec/tests/ContainerRowSerdeTest.cpp

Also, please, document this behavior in a method comment in .h file.

Reply:
I think https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/RowContainerTest.cpp#L281 was intended to test float comparison. But it didn't go as far as testing when these cases are in Arrays, Complex types. Might be good to enhance the tests here in RowContainer to check for the special floating point values in the Arrays, Maps and Complex types.